### ISSUE TITLE
Consistent 500 error code when map refreshes

### ISSUE THREAD

**Issue by bradleyshelley** [exodus4d/pathfinder#973]
<img width="545" alt="Annotation" src="https://user-images.githubusercontent.com/66140401/83299217-6396b980-a1bb-11ea-9d0d-833862b04d26.png">

Looking from some assistance here. It seems to happen more frequently when moving items on map.

**Comment by xrem** [comment]
@bradleyshelley 
Check your webserver error.log
It's kind of impossible to guess what happened without exception details and stacktrace.

**Comment by VyndarielAriami** [comment]
had the same issue, it seems that any part of the script is not written properly, as the memory usage is enormous.

Based on the cpu statics my new 16 core rig makes around 20 mOp/s (million operation per second) which I didnt even had while running gameservers on that machine.

Also the operations per second on hdd is quite high if the pathfinder is actively used.

With 1 member I have like 40 - 50 (read, as well as write) packets per seconds.

**Comment by ppfeufer** [comment]
@bradleyshelley :: Let me guess, you are using Chrome, right? I ask, because I observe the very same behavior for my Pathfinder Installation as well, but _only_ in Chrome.

Since there is nothing the any logs when this happens (apache error log, PHP error log, Pathfinder logs, all clear and nothing logged that even remotely hints towards anything) I kept my Chromes console open and running. and found this:

![Selection_359](https://user-images.githubusercontent.com/2989985/83360535-495b0800-a382-11ea-8c42-21d2b02c204e.jpg)

`net::ERR_NETWORK_CHANGED`, like WTF? That would mean I should be disconnected from more than just the Pathfinder website. But nope. EVE still running, Mumble didn't drop, Netflix (:-P) all fine. So my theory was, it has something to do with Chrome ... so I tested, for hours, with multiple browsers (Chrome, Firefox, Opera, Brave) all at the same time. The only browser that dropped me was, you guessed it, Chrome. (see next image: Firefox and Chrome side by side at the same time)

![Selection_358](https://user-images.githubusercontent.com/2989985/83360595-b7073400-a382-11ea-879f-41b05ecfbd36.jpg)

So ya, something's fishy in Chrome land I take it ....
